### PR TITLE
[PW-2778] - Display the payment methods as the default payment methods (PR2)

### PR DESCRIPTION
**Pull request by Morerice**
## Summary
Small css updated related to PR #169

### CODE REVIEW

File: views/templates/front/payment-method.tpl [hunk 1]
```diff
@@ -30,7 +30,7 @@
          data-local-payment-method="{$paymentMethodType|escape:'html':'UTF-8'}">
         {if $isPrestaShop16}
         <div class="col-xs-12 col-md-12">
-            <div class="payment_module" style="background-image: url(https://checkoutshopper-live.adyen.com/checkoutshopper/images/logos/medium/{$paymentMethodBrand|escape:'html':'UTF-8'}.png)">
+            <div class="payment_module adyen-collapse" style="background-image: url(https://checkoutshopper-live.adyen.com/checkoutshopper/images/logos/medium/{$paymentMethodBrand|escape:'html':'UTF-8'}.png)">
```
Comment (cyattilakiss) [inline views/templates/front/payment-method.tpl:33]: isn't it only added when `$collapsePayments`?

File: views/templates/front/stored-payment-method.tpl [hunk 2]
```diff
@@ -30,7 +30,7 @@
          data-stored-payment-api-id="{$storedPaymentApiId|escape:'html':'UTF-8'}">
         {if $isPrestaShop16}
             <div class="col-xs-12 col-md-12">
-                <div class="payment_module" style="background-image: url(https://checkoutshopper-live.adyen.com/checkoutshopper/images/logos/medium/{$logoBrand|escape:'html':'UTF-8'}.png)">
+                <div class="payment_module adyen-collapse" style="background-image: url(https://checkoutshopper-live.adyen.com/checkoutshopper/images/logos/medium/{$logoBrand|escape:'html':'UTF-8'}.png)">
```
Comment (cyattilakiss) [inline views/templates/front/stored-payment-method.tpl:33]: same here

File: views/templates/front/payment-method.tpl (hunk 1 above)
Comment (Morerice) [reply views/templates/front/payment-method.tpl:33]: The new class is just used to apply 1.6 styling to it, not to denote if the actual div is collapsible or not. Maybe renaming it to `adyen-collapse-styling` would make it more clear?

Comment (cyattilakiss) [reply views/templates/front/payment-method.tpl:33]: @Morerice yes I think it would be a bit better indeed

**Review by cyattilakiss — APPROVED** [review]

**Review by acampos1916 — APPROVED** [review]

**Pull request merged**